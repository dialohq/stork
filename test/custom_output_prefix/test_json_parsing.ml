let () =
  [ "custom_output_prefix_1.json"; "custom_output_prefix_bar.json" ]
  |> List.iter
       ~f:
         (Test_utils.print_parsed_json
            ~to_string:Custom_output_prefix.string_of_employee
            ~from_string:Custom_output_prefix.employee_of_string
            ~read_from_file:Custom_output_prefix.read_employee_from_file)
