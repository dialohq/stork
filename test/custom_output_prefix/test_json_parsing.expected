{"name":"Paul","age":30,"position":["dev",["Company",{"name":"Dialo"}]],"version":"bar","skills":[{"domain":"design","name":"typography"},{"domain":"computer science","name":"ocaml"}]}
---
{"name":"Paul","age":30,"position":["dev",["Company",{"name":"Dialo"}]],"version":"bar","skills":[{"domain":"design","name":"typography"},{"domain":"computer science","name":"ocaml"}]}
---
{"name":"Paul","age":30,"position":["dev",["Company",{"name":"Dialo","turnover":133700000}]],"version":"bar","skills":[{"domain":"design","name":"typography"},{"domain":"computer science","name":"ocaml"}]}
---
{"name":"Paul","age":30,"position":["dev",["Company",{"name":"Dialo","turnover":133700000}]],"version":"bar","skills":[{"domain":"design","name":"typography"},{"domain":"computer science","name":"ocaml"}]}
---
